staging/cert-manager-setup/templates/certificates.yaml

```diff
@@ -39,14 +39,16 @@ spec:
     - {{ . | quote -}}
   {{- end }}
 {{- end }}
-  keyAlgorithm: {{ .keyAlgorithm | default "rsa" }}
-  keyEncoding: {{ .keyEncoding | default "pkcs1" }}
-  keySize: {{ .keySize | default "2048" }}
+  privateKey:
+    algorithm: {{ .keyAlgorithm | default "RSA" }}
+    encoding: {{ .keyEncoding | default "PKCS1" }}
```

Comment:
out of api docs
>  allowed values are “pkcs1” and “pkcs8” standing for PKCS#1 and PKCS#8

are the docs wrong ? if so 🤦

Reply:
They are. Validation fails unless they are uppercase. It would have been nice if the enums contained both cases. I can convert them to uppercase to avoid validation issues with custom values.

Reply:
Can you please send a PR upstream to update docs?